src/main/java/org/folio/edge/sip2/repositories/PatronRepository.java

```diff
@@ -191,6 +200,28 @@ private Future<PatronInformationResponse> validPatron(String userId, Personal pe
         );
   }
 
+  private PatronInformationResponseBuilder populateChargedCount(JsonObject loans,
+                                      PatronInformationResponseBuilder builder) {
+    final int chargedItemsCount;
+    if (loans != null) {
+      chargedItemsCount = Math.min(getTotalRecords(loans), 9999);
```

Comment:
This is interesting. What is `Math.min` is doing for us here? Does it need a comment perhaps?

Reply:
Get minimum of total fine count & 9999 OR 
 Get minimum of total loan count & 9999